### DISCUSSION TITLE
Sign up - There seems to be no problem with the username [Hashmi_1]according to the rules provided by the github

### DISCUSSION THREAD

**Discussion by NumraHashmi** [community/community#110376] [New to GitHub]
![image](https://github.com/community/community/assets/114404101/6e0064c1-71cb-43b3-8ee0-ad0c6689cabf)

**Comment by queenofcorgis** [comment]
Hi @NumraHashmi , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.
